### PR TITLE
refactor: strapi multitenancy support

### PR DESCRIPTION
**Pull request by yildirayunlu**
Refactor a better multi tenancy experience with refine4.

### CODE REVIEW

File: examples/multi-tenancy-strapi/src/components/header/index.tsx [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+const { Text } = Typography;
+const { useToken } = theme;
+
+export const Header: React.FC<RefineThemedLayoutV2HeaderProps> = () => {
+    const { token } = useToken();
+
+    const authProvider = useActiveAuthProvider();
+    const { data: user } = useGetIdentity({
+        v3LegacyAuthProviderCompatible: Boolean(authProvider?.isLegacy),
```
Comment (alicanerdurmaz) [inline examples/multi-tenancy-strapi/src/components/header/index.tsx:16]: do we need this ?
```suggestion
-        v3LegacyAuthProviderCompatible: Boolean(authProvider?.isLegacy),
```

File: examples/multi-tenancy-strapi/src/components/header/index.tsx [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+                    display: "flex",
+                    justifyContent: "space-between",
+                }}
+            >
+                <StoreSelect />
+                <Space size="middle">
+                    {user?.username && (
+                        <>
+                            <Text strong>{user.username}</Text>
+                            <Avatar>R</Avatar>
```
Comment (alicanerdurmaz) [inline examples/multi-tenancy-strapi/src/components/header/index.tsx:45]: is this usage correct ? 🤔 I believe it should be like this or something to represent image `user.profileImage`.
```suggestion
 <Avatar>{user.username[0]}</Avatar>
```

**Review by alicanerdurmaz — CHANGES_REQUESTED** [review]

File: examples/multi-tenancy-strapi/src/components/header/index.tsx (hunk 2 above)
Comment (omeraplak) [reply examples/multi-tenancy-strapi/src/components/header/index.tsx:45]: <Avatar>{user.username?.[0]}</Avatar>

File: examples/multi-tenancy-strapi/src/components/header/index.tsx [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+                    display: "flex",
+                    justifyContent: "space-between",
+                }}
+            >
+                <StoreSelect />
+                <Space size="middle">
+                    {user?.username && (
+                        <>
+                            <Text strong>{user.username}</Text>
+                            <Avatar>{user.username[0].toUpperCase()}</Avatar>
```
Comment (omeraplak) [inline examples/multi-tenancy-strapi/src/components/header/index.tsx:41]:
```suggestion
                            <Avatar>{user.username?.[0].toUpperCase()}</Avatar>
```

**Review by alicanerdurmaz — APPROVED** [review]

**Pull request merged**